src/app/medical-record/medical-record.component.html

```diff
@@ -524,13 +550,45 @@ <h3 class="Color--warn Line--Break">Participant was withdrawn from the study!</h
       </tr>
       </tbody>
     </table>
+    </ng-container>
+    <ng-container *ngIf="source === 'bundle'">
+      <table class="table table-condensed">
+        <tbody>
+        <tr *ngFor="let pdf of pdfs">
+          <td>
+            <md-select placeholder="Select PDF" [(ngModel)]="pdf.order">
+              <md-option *ngFor="let item of [].constructor(pdfs.length + 1); let i = index" [value]="i">
+                {{i}}
+              </md-option>
+            </md-select>
+          </td>
+          <td>
+            {{pdf.displayName}}
+          </td>
+        </tr>
+        </tbody>
+      </table>
+      Order = 0 will not add the pdf to the bundle
+    </ng-container>
   </div>
   <div class="app-modal-footer">
-    <button type="button" class="btn btn-default"
-            (click)="modal.hide()">Close
-    </button>
-    <button type="button" class="btn btn-primary" [disabled]="participantExited"
-            (click)="downloadCoverPDFs()">Download Cover PDF
-    </button>
+    <ng-container *ngIf="source === 'cover'">
+      <div *ngIf="(medicalRecord.name == null || medicalRecord.name === '')"><b>* Institution name is needed for download</b></div>
```

Comment:
the same like `!medicalRecord.name` and parentheses are not necessary as well I guess